libpod/plugin/volume_api.go

```diff
@@ -153,6 +153,13 @@ func GetVolumePlugin(name string, path string) (*VolumePlugin, error) {
 	// And since we can reuse it, might as well cache it.
 	client := new(http.Client)
 	client.Timeout = defaultTimeout
+	// if the user specified a non-zero timeout, use their value. Else, keep the default.
+	if timeout != 0 {
+		if time.Duration(timeout)*time.Second < defaultTimeout {
+			logrus.Warn("the default timeout for volume creation is 5 seconds, setting a time less than that may break this feature.")
```

Comment:
```suggestion
			logrus.Warnf("the default timeout for volume creation is %d seconds, setting a time less than that may break this feature.", defaultTimeout)
```
Use defaultTimeout in case it changes in future.